emcc.py

```diff
@@ -2832,6 +2832,7 @@ def parse_string_list_members(text, sep):
     return result
 
   if text[0] == '[':
+    text = text.rstrip()
```

Comment:
Is this simpler as `text=text.strip()`, before the `if`?

Reply:
Maybe, is there ever a case where we'd care about whitespace?

Reply:
I'm the author of #6436. Sorry about letting this slip through.

If the arguments given is something of the form`"-s" "FOO=BAR   "`, then stripping beforehand would mean that the value would be parsed as `'BAR'` and not `'BAR   '`.  I don't know if that behaviour would ever be used, especially as the old parsing would have errored with that input.

Note also that this rstripping should probably also be included in https://github.com/kripken/emscripten/blob/7d09ea2eeb1fa424c1c5d96a7ab130f6a7de4e69/emcc.py#L2798-L2800

Reply:
No problem, hope the PR didn't come off accusatory! The code here is complicated so it's not surprising that bugs will pop up.

Reply:
BTW, fixed the case you described, and added a test. PTAL!